src/rpc/client/ts/runTests.ts

```diff
@@ -0,0 +1,69 @@
+const { execSync, spawn } = require('child_process')
+const { exit } = require('process')
+const glob = require('glob')
+const fs = require('fs')
+const os = require('os')
+const unzip = require('unzip-stream')
+
+// Extract server
+async function extractServer(filePath) {
+  await new Promise((res, rej) => {
+    let stream = fs
+      .createReadStream(`${filePath}.zip`)
+      .pipe(unzip.Extract({ path: '.' }))
+    stream.on('close', () => {
+      try {
+        res(`${filePath}.zip`)
+      } catch (err) {
+        rej(err)
+      }
+    })
+  })
+}
+
+// Run Scala gRPC server
+async function startServer(filePath) {
+  if (!os.platform().toLowerCase().startsWith('win')) {
+    execSync(`chmod +x ${filePath}/bin/omega-edit-grpc-server`)
+  }
+
+  let scriptName = os.platform().toLowerCase().startsWith('win')
+    ? `./${filePath}/bin/omega-edit-grpc-server.bat`
+    : `./${filePath}/bin/omega-edit-grpc-server`
+
+  const server_process = spawn(scriptName, [], {
+    stdio: 'ignore',
+    detached: true,
+  })
```

Comment:
Would be great if the port were configurable, even if through an environment variable, if going through yarn is awkward.

Reply:
@scholarsmate If you now do `PORT=9999 yarn test` it will bind to that port passed in otherwise defaults to 9000